leo/cli/commands/new.rs

```diff
@@ -39,9 +39,8 @@ impl Command for New {
 
     fn apply(self, context: Context, _: Self::Input) -> Result<Self::Output> {
         // Call the `aleo new` command from the Aleo SDK.
-        let command =
-            AleoNew::try_parse_from([ALEO_CLI_COMMAND, &self.name]).map_err(CliError::failed_to_parse_aleo_new)?;
-        let result = command.parse().map_err(CliError::failed_to_execute_aleo_new)?;
+        let command = AleoNew::try_parse_from([SNARKVM_COMMAND, &self.name]).map_err(CliError::failed_to_parse_new)?;
```

Comment:
Should this be renamed to `SnarkVMNew`? or something similar?
More generally, rename all occurrences of `aleo` to `snarkvm`?

Reply:
Will change to `SnarkVMNew`